api/envoy/extensions/filters/http/dynamic_modules/v3/dynamic_modules.proto

```diff
@@ -43,5 +45,5 @@ message DynamicModuleFilter {
   // For example, if a module has two filter implementations, one for logging and one for header manipulation,
   // filter_name is used to choose either logging or header manipulation. The filter_config can be used to
   // configure the logging level or the header manipulation behavior.
-  string filter_config = 3;
+  google.protobuf.Any filter_config = 3;
```

Comment:
could you add example yaml configurations here for string and bytes using `.. code-block:: yaml` like this?

https://github.com/envoyproxy/envoy/blob/cf8c6e76fb749e5111a9cdfe4c07992241496b9c/api/envoy/config/listener/v3/listener_components.proto#L274-L283

Reply:
bytes might be tough so only sting example would suffice :D